Makefile

```diff
@@ -3,8 +3,9 @@ MAKEFLAGS+=-r
 
 config=debug
 defines=standard
+cxxstd=default
```

Comment:
I thought `default` is the sanest option to make pugixml work anywhere (without any `-std=` option). If AppVeyor should build a C++11 build, can't we just say `cxxstd=c++11` somewhere in Appveyor.yml?